cdk/domino_cdk/provisioners/vpc.py

```diff
@@ -88,23 +140,46 @@ def provision_vpc(self, stack_name: str, vpc: config.VPC):
                 subnets=ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE),
             )
 
-    def provision_bastion(self, name: str, bastion: config.VPC.Bastion) -> None:
+        # TODO until https://github.com/aws/aws-cdk/issues/14194
+        for idx, subnet_id in enumerate(self.vpc.select_subnets(subnet_type=ec2.SubnetType.PUBLIC).subnet_ids):
+            name = f"DisableMapPublicIpLaunch-{idx}"
+            cr.AwsCustomResource(
+                self.scope,
+                name,
+                log_retention=logs.RetentionDays.ONE_DAY,
+                policy=cr.AwsCustomResourcePolicy.from_sdk_calls(resources=cr.AwsCustomResourcePolicy.ANY_RESOURCE),
+                on_create=cr.AwsSdkCall(
+                    action="modifySubnetAttribute",
+                    service="EC2",
+                    parameters={"MapPublicIpOnLaunch": {"Value": False}, "SubnetId": subnet_id},
+                    physical_resource_id=cr.PhysicalResourceId.of(name),
+                ),
+            )
+
+        if vpc.flow_logging:
+            if not flow_log_bucket:
```

Comment:
are 
* monitoring
* monitoring_bucket
* mb
* flow_log_bucket

mean the same thing? Could we name them the same?

Reply:
Yep!